### PR TITLE
fixing anchor link

### PR DESCRIPTION
**Pull request by zbtirrell**
See: https://central.tri.be/issues/80142

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$ticket' | [rsvp.php:84](https://github.com/moderntribe/event-tickets/blob/87f02a93ef31e589b2af1e06ceed761e86a731dd/src/views/tickets/rsvp.php#L84)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$ticket' | [rsvp.php:84](https://github.com/moderntribe/event-tickets/blob/87f02a93ef31e589b2af1e06ceed761e86a731dd/src/views/tickets/rsvp.php#L84)

> via jenkins/codesniffer-phpcs

File: src/views/tickets/rsvp.php [hunk 1]
```diff
@@ -21,177 +21,177 @@
...
-					<div class="tribe-rsvp-message tribe-rsvp-message-<?php echo esc_attr( $message->type ); ?>">
-						<?php echo esc_html( $message->message ); ?>
-					</div>
-					<?php
-				}//end foreach
-			}//end if
-			?>
+	<div class="tribe-rsvp-messages">
+		<?php
+		if ( $messages ) {
```
Comment (bordoni) [inline src/views/tickets/rsvp.php]: if this is a view; Can we make these tags more like the ones on the rest of our templates?
```php
<?php if ( condition ) : ?>
    <?php foreach ( $messages as $message ) : ?>
    <?php endforeach; ?>
<?php endif; ?>
```

Comment (zbtirrell) [inline src/views/tickets/rsvp.php]: no, that crushes my soul.

Comment (barryhughes) [inline src/views/tickets/rsvp.php]: Way to break Zach's heart, Gustavo

**Pull request merged**